cmd/crc/cmd/start.go

```diff
@@ -133,3 +137,23 @@ func getPullSecretFileContent() (string, error) {
 
 	return pullsecret, nil
 }
+
+func checkIfNewVersionAvailable(noUpdateCheck bool) {
+	if noUpdateCheck {
+		return
+	}
+	newVersionAvailable, newVersion, err := version.NewVersionAvailable()
+	if err != nil {
+		logging.Debugf("Error checking if a new version is available: %v", err)
+		return
+	}
+	if newVersionAvailable {
+		logging.Infof("A new version (%s) has been published", newVersion)
+		userWantsToContinue := input.PromptUserForYesOrNo("Would you like to continue using the current version", false)
+		if !userWantsToContinue {
+			errors.ExitWithMessage(0, fmt.Sprintf("Please download the new version from %s", constants.DefaultPullSecretURL))
+		}
```

Comment:
We'd only prompt when a new version is released, and at that point the user should actually update since the certs are also likely to expire, also a flag to start command `--no-update-check` is added, and similar config option to disable this, we could mention about this in the message. Do you still think we should not show the prompt and just a warning?

Reply:
Well, once a new version is released, we'll always prompt. And maybe the certs will expire soon, maybe not, maybe we made a minor release to fix a critical bug on a platform you are not using. In any case, a prompt saying "Do you want to abort?" is odd.

Reply:
So, a `WARN` would be enough here? cc @code-ready/crc-devel